spec/models/students_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Student, type: :model do
+  describe 'validations' do
+    it { should validate_presence_of :first_name }
+    it { should validate_presence_of :last_name }
+    it { should validate_presence_of :cohort_id }
+    it { should validate_presence_of :email }
+    it { should validate_presence_of :alumni }
+  end
+
+  describe 'relationships' do
+    it { should belong_to :cohort }
+  end
+end
```

Comment:
Excellent. This should be what we need to start. It will be interesting to see how Devise + OmniAuth + GitHub OAuth work together will this account information.